File: Java/Canica.java
public class Canica {
   private String tamano;
   private float peso;

   public String getTamano() {
      return tamano;
   }

   public float getPeso() {
      return peso;
   }

   public void setTamano(String tamano) {
      this.tamano = tamano;
   }

   public void setPeso(float peso) {
      this.peso = peso;
   }

   public static void main(String[] args) {
      Canica canica1 = new Canica();

      String tamano = canica1.getTamano();
      float peso = canica1.getPeso();
      System.out.println("El estado actual del objeto es: ");
      System.out.println("El tamano del atributo tamano de canica1 es: " + tamano);
      System.out.println("El tamano del atributo peso de canica1 es: " + peso);

      String tamano1 = "Grande";
      float peso1 = (float) 12.3;
      canica1.setTamano(tamano1);
      canica1.setPeso(peso1);

      System.out.println("El nuevo estado actual del objeto es: ");
      System.out.println("El nuevo tamano del atributo tamano de canica1 es: " + canica1.getTamano());
      System.out.println("El nuevo tamano del atributo peso de canica1 es: " + canica1.getPeso());
   }

}

File: Python/Canica.py
class Canica():
    __tamano = None
    __peso = None

    def getTamano(self):
        return self.__tamano

    def getPeso(self):
        return self.__peso

    def setTamano(self, tamano):
        self.__tamano = tamano

    def setPeso(self, peso):
        self.__peso = peso


def main():
    miCanica = Canica()
    print("EL estado actual del objeto es: ")
    print("EL tamano del atributo tamano de miCanica es: ", miCanica.getTamano(
    ), "\nEl peso de atributo peso de miCanica es: ", miCanica.getPeso())

    miCanica2 = Canica()
    tamano2 = "Grande"
    peso2 = 12.3
    miCanica2.setTamano(tamano2)
    miCanica2.setPeso(peso2)
    print("EL estado actual del objeto es: ")
    print("EL tamano del atributo tamano de miCanica2 es: ", miCanica2.getTamano(
    ), "\nEl peso de atributo peso de miCanica2 es: ", miCanica2.getPeso())


if __name__ == '__main__':
    main()
